src/NServiceBus.Core.Tests/API/APIApprovals.ApproveNServiceBus.approved.txt

```diff
@@ -311,6 +311,9 @@ namespace NServiceBus
         public void ExcludeTypes(params System.Type[] types) { }
         public void OverridePublicReturnAddress(string address) { }
         public void RegisterComponents(System.Action<NServiceBus.ObjectBuilder.IConfigureComponents> registration) { }
+        [System.ObsoleteAttribute("Implement `IWantToRunWhenEndpointStartsAndStops` in the NServiceBus.Host package " +
```

Comment:
There seems to be 2 use cases here:

1. User is selfhosting => should just call whatever code he needs before/after the call to Enpoint.Create/Start and also before/after `endpointInstance.Stop`.
2. User is using one of the hosts => just use the new interface.

Do we need to spell this out in the obsolete message?